capabilities/connect.capabilities.yml

```diff
@@ -173,6 +174,9 @@ properties:
       - System-SwUpdate
       - System-SwUpdate-Partial
       - Port-Mirroring
+      # Security
+      - MAC-
```

Comment:
Not sure what this shoul've been?
MAC- seems invalid

Reply:
Agreed, should not end with '-'

Reply:
Its supposed to be MAC-ACL, I stopped typing it looks like ;-) Correcting it..